core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/LruBlockCache.java

```diff
@@ -84,7 +85,7 @@
   static final int statThreadPeriod = 60;
 
   /** Concurrent map (the cache) */
-  private final ConcurrentHashMap<String,CachedBlock> map;
```

Comment:
If have a CacheType, then could possibly deprecate the `tserver.cache.data.size` type settings in favor of every implementation using `general.custom.xyz.` settings.  Or we could assume that every implementation will at least want a max size and keep these settings around.  I don't know whats best.